examples/apps/helloworld/app.py

```diff
@@ -16,11 +16,22 @@
 import json
 from flask import Flask, request
 app = Flask(__name__)
+version = ""
 
 @app.route('/hello')
 def hello():
-    service_version = os.environ.get('A8_SERVICE').split(':')
-    version = service_version[1] if len(service_version) == 2 else 'UNVERSIONED'
+    global version
```

Comment:
not that efficiency is a concern, but no point in reading and parsing env var on every request.
That's as far as my Python skills go ;-)